CWE/events/italy.txt

```diff
@@ -2756,7 +2756,7 @@ country_event = {
 	trigger = {
 		tag = ITA
 		year = 1972 
-		NOT = { year = 1978 }
+		NOT = { year = 1973 }
```

Comment:
Please try not to keep the start and end dates too close to each other as sometimes the MTTH doesn't work and the event fires one year later. This is usually when the player saves his game in 1972 to continue later.